ads/google/doubleclick.js

```diff
@@ -100,14 +100,12 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     const primaryWidth = primarySize[0];
     const primaryHeight = primarySize[1];
     // By default, this flag is true.
-    const multiSizeValidation = data.multiSizeValidation ?
-        data.multiSizeValidation == 'true' : true;
 
     getMultiSizeDimensions(
         multiSizeDataStr,
         primaryWidth,
         primaryHeight,
-        multiSizeValidation,
+        (data.multiSizeValidation || 'true') == 'true',
```

Comment:
whoops looks like I'm the one low on caffeine. leave it as is, you're right. I wish there was a prettier way to write that but I guess that's it